components/popover/tippy_utils.js

```diff
@@ -13,6 +13,13 @@ export const TIPPY_STICKY_VALUES = [true, false, 'reference', 'popper'];
 export const createTippy = (anchorElement, options) => {
   const { contentElement } = { ...options };
   delete options.contentElement;
+  if (contentElement === null) {
```

Comment:
i would probably not use the utility and use tippy directly but let me know if this is prefered.

Reply:
Yeah i kinda figured what was in there might not work for your case.

Reply:
i will remove this code and directly use tippy

Reply:
> i will remove this code and directly use tippy

It seems like you were going to remove this code but the code is still there... whats the reason?

Reply:
removed it now! i didnt push the commit my bad!